salt/modules/zabbix.py

```diff
@@ -1430,7 +1430,7 @@ def host_inventory_get(hostids, **connection_args):
             ret = _query(method, params, conn_args["url"], conn_args["auth"])
             return (
                 ret["result"][0]["inventory"]
-                if len(ret["result"][0]["inventory"]) > 0
+                if ret["result"] and ret["result"][0]["inventory"]
```

Comment:
**comment**: I don't think it reads any better, but it *could* be written:

```
return next(iter(ret.get("result") or [{"inventory": False}])).get("inventory") or False
```

(or, something like that. But, like I said, it's not exactly an improvement :trollface: )

Reply:
Oh, of course I have a bias here, but this one I think my version is easier to understand.

Reply:
I like "easier to understand"